### ISSUE TITLE
[BUG] A11y | Missing focus styles in many components

### ISSUE THREAD

**Issue by pixelass** [DivanteLtd/storefront-ui#588]
**Describe the bug**
A lot of elements don't have proper accessibility support. There are no styles for focus which prevents keyboard access.

**How to reproduce**
1. Go to https://storybook.storefrontui.io/?path=/story/molecules-pagination--common (as example, many elements/components have the same issue)
2. Interact with the pagers
3. Use tab 
4. No focus styles are ever applied

**Expected behavior**
I should be able to see which element is focused

**Actual behavior**
There are no focus styles at all.

**Comment by pixelass** [comment]
Example: Button.

https://storybook.storefrontui.io/?path=/story/atoms-button--common

The following code prevents the focus ring:

```css
sf-button {
    outline: 0;
}
```

**Comment by filrak** [comment]
Hey, thanks for pointing out! Is it something that you would like to fix or just report ;) If just report I will look internally for someone to resolve it

**Comment by pixelass** [comment]
I am very open to help since this seems to be an issue in several components.

**Comment by pixelass** [comment]
Is there a predefined style for focus rings?

**Comment by aniamusial** [comment]
@pixelass we are thinking of using the hover as focus styles, but maybe you have a better idea? We are open for any suggestions :)

**Comment by Michal-Dziedzinski** [comment]
@aniamusial imo using hover styles as focus styles could be not a good idea, if somebody is color-blind he may not recognize that somethig is focused. We need to think about something else.

**Comment by pixelass** [comment]
I think the best option would be to use the native focus or make use of the color `highlight`.
This way the users OS preferences are incorporated.

There are several techniques to make this work.

I only skimmed this article but seems to cover a cross browser attempt: https://ghinda.net/article/mimic-native-focus-css/

**Comment by pixelass** [comment]
Sometimes I like to make the focus-ring a component, so I can just reuse it and don't have to worry about it. I can just add the ring where I see fit and assign a variable to show or hide it. 
Since it's only a decorative/a11y component I also apply `pointer-events: none` to it, which makes it even more flexible.

**Comment by aniamusial** [comment]
@pixelass Thank you! Your proposed solutions sounds good - feel free to take care of that issue :)

**Comment by pixelass** [comment]
awesome, I'll see what I can do.

**Comment by pixelass** [comment]
I was moved to a different project at work. It's very unlikely for me to find time to implement this. Sorry.

**Comment by aniamusial** [comment]
@pixelass Sure, totally understandable :) Thank you for all the help!
